build.gradle

```diff
@@ -76,5 +81,13 @@ wrapper {
 }
 
 dependencyCheck {
-    suppressionFile = "${project.rootDir}/dependency-check-suppressions.xml"
+    suppressionFile = '${project.rootDir}/dependency-check-suppressions.xml'
```

Comment:
add nvd api key, then it should run faster (~ 5-7 mins)
`nvd {
            apiKey = findProperty('NVD_API_KEY') ?: System.getenv('NVD_API_KEY') ?: "FAKE_DC_NVD_API_KEY"
            validForHours = 50
            delay = 4000
        }`

Reply:
NVD_API_KEY is env variable and you need to add it to pipe in CI (on the github side)

Reply:
Added NVD API KEY, dependency analyzer works for more than 10 minutes